.babelrc.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         },
       },
     ],
-    '@babel/preset-stage-2',
+    ["@babel/preset-stage-2", { "decoratorsLegacy": true }],
```

Comment:
I forget why this was added?

Reply:
It isn't necessary for our version of Babel (if we ever upgrade while it's still in beta then `decoratorsLegacy` will become something we have to set explicitly). I can remove it.